### ISSUE TITLE
The Documentation for Service Account Keys is A Bit Outdated

### ISSUE THREAD

**Issue by kvnkho** [PrefectHQ/ui#997]
**What**

There is a tutorial [here](https://cloud.prefect.io/tutorial/Universal-Deploy#authenticating-with-prefect-cloud) in the UI that is a bit outdated when it comes to service keys It still says tokens in some areas.

We may even want this tutorial removed and point them in the docs so we don't have to keep this in two places.

* [ ] Remove or update the tutorial section.
* [ ] Remove it from the NavBar also

<img width="1436" alt="Screen Shot 2021-08-11 at 3 54 38 PM" src="https://user-images.githubusercontent.com/32503212/129094769-5dc116ed-bbc1-43c6-ab10-1f84db33700d.png">

**Comment by sethschori** [comment]
As someone who was just using this now, I agree that it might have some outdated info in it. What was really helpful for me as a brand new user was when I was on the Cloud UI's Dashboard page it said something like this:

> Psst! Looks like you don't have any Flows yet. Follow this tutorial ...

(I don't remember the exact wording and it's no longer appearing since I made my first test flow.)

So, I would advocate for keeping that "Psst" message and pointing it to wherever the new home is in the docs, since I found the beginner tutorial really helpful for figuring out how to get started, even if some of the info was outdated and a bit confusing.

TLDR; Please make sure there's a link to wherever the tutorial will live in the documentation -- and don't get rid of all beginner tutorials, please, since I would have had a much harder time figuring out how to get started without one.
